Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/UserController.cs

```diff
@@ -7,10 +7,9 @@
 using Microsoft.AspNetCore.Identity;
 using Promact.Oauth.Server.Models.ApplicationClasses;
 using System.Threading.Tasks;
-using System.Linq;
+using Exceptionless;
```

Comment:
I am not sure why have you added Exceptionless here in using. But it serves no purpose. Please remove all unused usings